src/main/resources/application.yaml

```diff
@@ -249,9 +249,11 @@ darts:
         unstructured: darts-unstructured
         inbound: darts-inbound-container
         outbound: darts-outbound
+        arm: dropzone
       sas-url:
         darts-inbound-container: ${DARTS_INBOUND_STORAGE_SAS_URL}
         darts-unstructured: ${DARTS_UNSTRUCTURED_STORAGE_SAS_URL}
+        dropzone: ${ARM_SAS_ENDPOINT}
```

Comment:
why are we using a new attribute for a SAS URL , I thought we already have one "sas-endpoint: ${ARM_SAS_ENDPOINT:}"

Reply:
because the existing service we are re-using, `DataManagementService.copyBlobData`, expects sas urls to be in  `darts.storage.blob.sas-url.<container_name>`. See [here](https://github.com/hmcts/darts-api/blob/130362d6cf9161077b32af38323eebb39156f5c5/src/main/java/uk/gov/hmcts/darts/datamanagement/config/DataManagementConfiguration.java#L61).
I didn't want to change the existing code and add a special exception for arm, therefore making it less generic.